tests/ut/conftest.py

```diff
@@ -61,11 +59,13 @@ def base_cache():
 
 @pytest.fixture
 async def redis_cache():
-    async with RedisCache() as cache:
+    redis = pytest.importorskip("aiocache.backends.redis")
+    async with redis.RedisCache() as cache:
         yield cache
 
 
 @pytest.fixture
 async def memcached_cache():
-    async with MemcachedCache() as cache:
+    memcached = pytest.importorskip("aiocache.backends.memcached")
+    async with memcached.MemcachedCache() as cache:
```

Comment:
....and this, are more problematic.

`pytest.importorskip()` returns `Any`, which loses all our type checking features when we check with mypy. We use this to help verify that users won't encounter type errors when using the library correctly (as tests/examples are the only place we actually test/use the public API).

Reply:
Ah, this never occurred to me! I suppose we could either do something like:

    pytest.importorskip(...)  # just to skip test if it's missing
    from ... import ...  # the actual import

or:

    try:
        from ... import ...
    except ImportError:
        pytest.skip("x is missing")

Which one would you prefer?

Reply:
I think the first is preferable.